lib/rabbitmq/cli/command_behaviour.ex

```diff
@@ -20,16 +20,20 @@ defmodule RabbitMQ.CLI.CommandBehaviour do
   @callback merge_defaults(List.t, Map.t) :: {List.t, Map.t}
   @callback banner(List.t, Map.t) :: String.t
   @callback run(List.t, Map.t) :: any
-  @callback switches() :: Keyword.t
-  @callback aliases() :: Keyword.t
   # Coerces run/2 return value into the standard command output form
   # that is then formatted, printed and returned as an exit code.
   # There is a default implementation for this callback in DefaultOutput module
   @callback output(any, Map.t) :: :ok | {:ok, any} | {:stream, Enum.t} |
                                   {:error, ExitCodes.exit_code, [String.t]}
   @optional_callbacks formatter: 0,
                       scopes: 0,
-                      usage_additional: 0
+                      usage_additional: 0,
```

Comment:
I guess the point of "default implementation" means that a behaviour could also define a function to fall back to but that wouldn't help us avoid `function_exported?/4` entirely.